scripts/utils/ttGen.ps1

```diff
@@ -0,0 +1,12 @@
+Write-Host "Regenerating all generated files from all .tt files."
+Write-Host "(requires t4.exe from 'dotnet tool install --global dotnet-t4' or right click .tt files in VS and 'run custom tool')"
+Write-Host ""
+$ttFiles = Get-ChildItem -Filter *.tt -Recurse -File -Name
+foreach ($ttFile in $ttFiles)
+{
+    if (-not $ttFile.endswith("AssemblySourceFileGenerator.tt"))
+    {
+        Write-Host t4.exe $ttFile
+        t4.exe "$ttFile"
+    }
+}
```

Comment:
our repo's `.editorconfig` file specifies `insert_final_newline = true`, meaning your editor doesn't respect `.editorconfig` :(

Reply:
reading https://github.com/PowerShell/PowerShellEditorServices/issues/665 it seems that i needed to install the "editorconfig for vs code" extension. It then added an trailing newline. will commit change too.